identity/webapp/pages/registration.tsx

```diff
@@ -49,18 +49,20 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
 
     try {
       const token = decodeToken(sessionToken);
+      console.log({ token });
 
       if (typeof token !== 'string') {
         email = token.email;
       }
     } catch {
+      console.log('no valid token');
       // Redirect if we don't have a valid token from the Auth0 signup action
-      return {
-        redirect: {
-          destination: '/',
-          permanent: false,
-        },
-      };
+      // return {
+      //   redirect: {
+      //     destination: '/',
+      //     permanent: false,
+      //   },
+      // };
```

Comment:
Why is this redirect commented out and not deleted?

Reply:
I was just catching up with David about this, I think we don't want to return a redirect in a catch, I reckon we'll want to throw and error and render that on the page in some way.

Reply:
we could redirect to `/error`?

Reply:
Let's see if this is where the issue is. But I think we will ultimately need to catch and surface the error